pkg/connector/user.go

```diff
@@ -193,13 +278,14 @@ func ciamUserBuilder(domain string, apiToken string, client *okta.Client, emailF
 	}
 }
 
-func userBuilder(domain string, apiToken string, client *okta.Client, skipSecondaryEmails bool) *userResourceType {
+func userBuilder(domain string, apiToken string, client *okta.Client, skipSecondaryEmails bool, connector *Okta) *userResourceType {
```

Comment:
At this point it's probably worth changing this function signature to only take the connector. All the other arguments are attributes of it.